stream-chat-android-client/src/main/java/io/getstream/chat/android/client/utils/ChatUtils.kt

```diff
@@ -3,9 +3,9 @@ package io.getstream.chat.android.client.utils
 import android.util.Base64
 import java.nio.charset.StandardCharsets
 
-public object ChatUtils {
+internal object ChatUtils {
```

Comment:
We could just deprecate this class at first, instead of breaking public API immediately

Reply:
This method is rarelly used (Only works for development apps, where fixed some weeks ago and we didn't receive any feedback about it), We don't need to deprecate it

Reply:
In that case, can we just remove this file and move its contents into the client method?